packs/journals/classes.json

```diff
@@ -182,7 +182,7 @@
r reach by an additional 10 feet, instead of the usual additional 5 feet.</li><li><strong>Impossible Alloy</strong> Other inventors claim it's not even technically possible, but you've managed to create several metal alloys that seem to work for only you. These alloys can damage opponents vulnerable to any one of the seven skymetals. Your innovation is treated as all seven skymetals (abysium, adamantine, djezet, inubrix, noqual, orichalcum, and siccatite). This means you deal more damage to a variety of creatures, though you don't apply any of the other special effects for weapons made of those skymetals.</li><li><strong>Momentum Retainer (Melee only)</strong> A special weighted device lets your weapon retain more of its momentum when you attack. Your innovation gains the forceful and versatile B traits.</li><li><strong>Omnirange Stabilizers (Ranged only)</strong> You've modified your innovation to be dangerous and effective at any range. If your innovation had the volley trait, remove the volley trait. Otherwise, increase your innovation's range increment by 50 feet or an amount equal to the weapon's base range increment, whichever is more.</li><li><strong>Rune Capacity</strong> Whether you've dabbled with orichalcum or found another engineering solution, you've built your innovation in such a way that it can hold an additional property rune. Your innovation can have one more property rune than a normal item of its kind (to a maximum of four property runes with a +3 potency weapon).</li></ul><h2>Inventive Mastery<span style=\"float:right\">Level 17</span></h2><p>Your inventions are incredibly effective. Your proficiency rank for your inventor class DC increases to master.</p><h2>Juggernaut<span style=\"float:right\">Level 17</span></h2><p>Your body is accustomed to physical hardship and resistant to ailments. Your proficiency rank for Fortitude saves increases to master. When you roll a success on a Fortitude save, you get a critical success instead.</p><h2>Infinite Invention<span style=\"float:right\">Level 19</span></h2><p>Your ability to adjust your innovation has reached impossible heights, and you can use these skills to make major adjustments in your spare time. During your daily preparations, you automatically fix your innovation if it's destroyed or broken, and you can change to a different innovation (armor, construct, weapon, or other type you have access to) and change your modifications and offensive boost.</p><p>If you have any feats that had your previous innovation as a prerequisite, you can't use them until you retrain the feats as normal. However, it takes you only 1 day of downtime to retrain such a feat into a feat that has your new innovation as a prerequisite, instead of 1 week.</p><h2>Medium Armor Mastery<span style=\"float:right\">Level 19</span></h2><p>Your skill with armor improves, helping you avoid more blows. Your proficiency ranks for light and medium armor, as well as for unarmored defense, increase to master.</p>",
```

Comment:
"you boost four different ability scores" to "you boost four different attributes", etc. :)